src/styles/about.scss

```diff
@@ -0,0 +1,98 @@
+.about {
+  font-size: 14px;
```

Comment:
The font-size and font-family are not defined here. As per the task, you need to use 'Sharp Sans No2 Medium' and 'Avenir' fonts. Make sure you are importing and using the right fonts from the fonts directory.